Algorithms/Mathematical-Algorithms/Power/Power_in_your_Templete.cpp

```diff
@@ -0,0 +1,32 @@
+//https://github.com/sahilbansal17/Competitive_Coding/
+
+//Name: Anil Khatri (@imkaka)
+//Email-Id: anil.soccer.khatri@gmail.com
+
+// Now we understand the Algorithm in other file we need to have it in our Templete to use Power()
```

Comment:
Fix the typo ~templete~ to template wherever mistyped.

Reply:
Ok Thanks